README.md

```diff
@@ -13,33 +13,30 @@
 
 # PyGridSim
 
-Package to simulate OpenDSS circuits on Python.
-
-NOTE: README comes from dai cookie cutter, will be updated
-NOTE: will be moved to Dai lab repository
+PyGridSim is a package that allows user to simulating OpenDSS circuits with Python. PyGridSim uses a functional interface to allow users to efficiently generate circuits of various scopes.
```

Comment:
PyGridSim is a Python package for simulating OpenDSS circuits. PyGridSim uses a functional interface to allow users to efficiently generate circuits of various scopes.